applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -64,6 +64,27 @@ For the USB legacy stack, this results in updating defaults for the following Kc
 
 For the USB next stack, the module uses dedicated APIs to set the USB device identifiers during initialization.
 
+.. _nrf_desktop_usb_state_sof_synchronization:
+
+USB Start of Frame (SOF) synchronization
+========================================
+
+The module receives a HID input report as :c:struct:`hid_report_event` and submits the report to USB stack.
+The module informs that the HID report was sent using :c:struct:`hid_report_sent_event`.
+
+* If the :ref:`CONFIG_DESKTOP_USB_HID_REPORT_SENT_ON_SOF <config_desktop_app_options>` Kconfig option is disabled, the :c:struct:`hid_report_sent_event` is instantly submitted when HID report is sent over USB during USB poll (on USB endpoint read).
+  This approach results in shorter HID data lantency as HID report pipeline is not used.
+  However, it may lead to no HID report provided by the USB peripheral during an USB poll if two subsequent USB polls for HID data happen in a quick succession.
+  USB polls for HID data are not guaranteed to be evenly spaced in time.
+  * If the :ref:`CONFIG_DESKTOP_USB_HID_REPORT_SENT_ON_SOF <config_desktop_app_options>` Kconfig option is enabled, submitting :c:struct:`hid_report_sent_event` is delayed until subsequent USB SOF is reported by the USB stack.
+  USB SOFs, in contrast to USB polls, are always evenly spaced in time.
+  Enabling the Kconfig option allows to avoid negative impact of jitter related to USB polls and ensures that HID peripheral can provide HID input report during every USB poll.
+  However, the feature also increases HID data latency as HID report pipeline with length of two is used.
+  Without the pipeline, an USB poll happening quickly after an USB SOF would result in no HID report provided by the peripheral because the HID report source would be unable to provide subsequent HID input report in time.
+
+  In case of :ref:`nrf_desktop_hid_mouse_report_handling`, enabling the USB SOF synchronization also synchronizes motion sensor sampling to the USB SOF instead of USB polls (motion sensor sampling is synchronized to :c:struct:`hid_report_sent_event`).
+  That ensures sampling the sensor more evenly.
```

Comment:
```suggestion
USB Start of Frame (SOF) synchronization
========================================

The module receives an HID input report as :c:struct:`hid_report_event` and submits the report to the USB stack.
The module informs that the HID report was sent using :c:struct:`hid_report_sent_event`.

* If the :ref:`CONFIG_DESKTOP_USB_HID_REPORT_SENT_ON_SOF <config_desktop_app_options>` Kconfig option is disabled, the :c:struct:`hid_report_sent_event` is instantly submitted when HID report is sent over USB during USB poll (on USB endpoint read).
  This approach results in shorter HID data latency as an HID report pipeline is not used.
  However, it may lead to no HID report provided by the USB peripheral during an USB poll if two subsequent USB polls for HID data happen in quick succession.
  USB polls for HID data are not guaranteed to be evenly spaced in time.
  * If the :ref:`CONFIG_DESKTOP_USB_HID_REPORT_SENT_ON_SOF <config_desktop_app_options>` Kconfig option is enabled, submitting :c:struct:`hid_report_sent_event` is delayed until subsequent USB SOF is reported by the USB stack.
  USB SOFs, in contrast to USB polls, are always evenly spaced in time.
  Enabling the Kconfig option removes the negative impact of jitter related to USB polls and ensures that an HID peripheral can provide an HID input report during every USB poll.
  However, the feature also increases HID data latency as an HID report pipeline with a length of two is used.
  Without the pipeline, a USB poll happening quickly after a USB SOF would result in no HID report provided by the peripheral because the HID report source would be unable to provide a subsequent HID input report in time.

  In the case of :ref:`nrf_desktop_hid_mouse_report_handling`, enabling the USB SOF synchronization also synchronizes motion sensor sampling to the USB SOF instead of USB polls (motion sensor sampling is synchronized to :c:struct:`hid_report_sent_event`).
  This synchronization ensures that the sensor is sampled more evenly.
```

Reply:
if `remove` is too strong in context of jitter, you can use `minimize` or `reduce` verbs

Reply:
I think we can call it `remove` here (we no longer rely on USB polls so that negative impact of jitter specific to the USB polls is actually removed)

Reply:
Let's use `reduce` term here (the USB polling accuracy still affects HID data latency in some way)